code/style.css

```diff
@@ -6,14 +6,14 @@ body {
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  background: #99cc33;
 }
 
 h1 {
   font-weight: bold;
   font-size: 44px;
   line-height: 34px;
-  color: #fff;
+  color: #ff3399;
   text-align: center;
 }
```

Comment:
I can't comment on the correct line but I would suggest getting rid of the `width` of your images to remove the distortion :) Compare image 1 and 2 👇
<img width="840" alt="Skärmavbild 2021-01-29 kl  16 41 51" src="https://user-images.githubusercontent.com/51529201/106295559-e4c5b500-6250-11eb-962e-e6876e5fd743.png">
<img width="822" alt="Skärmavbild 2021-01-29 kl  16 42 07" src="https://user-images.githubusercontent.com/51529201/106295592-ee4f1d00-6250-11eb-961f-f0f397f8ec43.png">